kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/push/NotifyingPushConnection.java

```diff
@@ -71,6 +84,10 @@ private void notifyPushListeners(Consumer<PushSendListener> action) {
         }
     }
 
+    private Logger getLogger() {
+        return LoggerFactory.getLogger(getClass());
+    }
```

Comment:
Can we initialize the logger as a static field instead?

Reply:
done